### PR TITLE
German translation of the doc

### PR DESCRIPTION
**Pull request by SvA1**
Issue #1

### CODE REVIEW

**Comment by nlepage** [comment]
Thanks, I'll try to review today.

File: docs/README_de.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+// ➔ "1000000 ist many and fällt zurück auf other"
+```
+
+Einige Sprachen können Standard-Fallbacks haben, siehe [locales.ts](https://github.com/nlepage/templural/blob/main/src/locales.ts).
+
+## ❓ FAQ
+
+### Was ist mit negativen Zahlen, Gleitkommazahlen oder anderen Werten?
+
+templural kümmert sich nicht darum, es verlässt sich auf [Intl.PluralRegeln](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules).
```
Comment (nlepage) [inline docs/README_de.md:302]:
```suggestion
templural kümmert sich nicht darum, es verlässt sich auf [Intl.PluralRules](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules).
```

File: docs/README_de.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+// n = 1000000 ➔ "1000000 ist in der Kategorie many"
+// n = 0       ➔ "0 ist in der Kategorie one"
+```
+Siehe [Pluralregeln der Sprachen](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html) für Informationen über Pluralregeln in jeder Sprache.
+
+Das Verhalten von templural kann über drei Mechanismen angepasst werden:
+
+ - [Kategoriepriorität](#category-priority)
+ - [Kategoriereihenfolge](#category-order)
+ - [Kategorie Fallback](#category-fallback)
```
Comment (nlepage) [inline docs/README_de.md:206]:
```suggestion
 - [Kategoriepriorität](#kategoriepriorit%C3%A4t)
 - [Kategoriereihenfolge](#kategoriereihenfolge)
 - [Kategorie Fallback](#kategorie-fallback)
```

File: docs/README_de.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+Ein besonderer Dank geht an [Valentin Cocaud](https://github.com/EmrysMyrddin) und [Yoann Prot](https://github.com/Taranys) für ihre ursprüngliche Idee von templural.
+
+👤 **Nicolas Lepage**
+
+* Twitter: [@njblepage](https://twitter.com/njblepage)
+* GitHub: [@nlepage](https://github.com/nlepage)
+
+👩 **SvA1**
+
+* GitHub: [@SvA1](https://github.com/SvA1)
```
Comment (nlepage) [inline docs/README_de.md:325]: Sorry I've refactored this section.

There are a few things that cannot be translated:
 - the section's title must be "Contributors ✨"
 - the HTML between the comments

However you can translate the two sentences:
 - Thanks goes to these wonderful people
 - This project follows...

```suggestion
## Contributors ✨

<table>
  <tr>
    <td align="center"><a href="https://github.com/SvA1"><img src="https://avatars.githubusercontent.com/u/19571875?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Nicolas Lepage</b></sub></a><br /><a href="https://github.com/nlepage/templural/commits?author=nlepage" title="Author">✍️</a></td>
  </tr>
</table>

Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):

<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
<!-- prettier-ignore-start -->
<!-- markdownlint-disable -->
<table>
  <tr>
    <td align="center"><a href="https://github.com/EmrysMyrddin"><img src="https://avatars.githubusercontent.com/u/3855602?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Valentin Cocaud</b></sub></a><br /><a href="#ideas-EmrysMyrddin" title="Ideas, Planning, & Feedback">🤔</a></td>
    <td align="center"><a href="https://github.com/Taranys"><img src="https://avatars.githubusercontent.com/u/4621525?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Yoann Prot</b></sub></a><br /><a href="#ideas-Taranys" title="Ideas, Planning, & Feedback">🤔</a></td>
    <td align="center"><a href="https://github.com/SvA1"><img src="https://avatars.githubusercontent.com/u/41849274?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Svetlana A.</b></sub></a><br /><a href="https://github.com/nlepage/templural/commits?author=SvA1" title="Documentation">📖</a></td>
  </tr>
</table>

<!-- markdownlint-restore -->
<!-- prettier-ignore-end -->

<!-- ALL-CONTRIBUTORS-LIST:END -->

This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
```

File: docs/README_de.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+// nbIdeen = 1  ➔ "Yoann und Valentin hatten 1 interessante Idee"
+// nbIdeen = 2  ➔ "Yoann and Valentin hatten 2 interessante Ideen"
+// nbIdeen = 42 ➔ "Yoann und Valentin hatten 42 interessante Ideen"
+// nbIdeen = 0  ➔ "Yoann and Valentin hatten 0 interessante Ideen"
+```
+
+### Beliebiger anderer Werte in den Text einfügen
+
+```js
+templural`${userName} hat ${nbPunkten} Punkt{e}`
```
[the remark this answers is not in the document]
Comment (SvA1) [reply docs/README_de.md:81]: I'll change it to **benutzerName**

**Comment by SvA1** [comment]
Here I've added a spelling mistake in the plural sentences, is it what was needed?
```
// nbBiere = 1 ➔ "Ich hatte nur 1 Bier, Liebling, ich schwöre"
// nbBiere = 2 ➔ "Ich hatte nur 2 Bären, Liebling, ich schwöre"
// nbBiere = 6 ➔ "Ich hatte nur 6 Bären, Liebling, ich schwöre"
```

**Comment by SvA1** [comment]
> 
> 
> > Here I've added a spelling mistake in the plural sentences, is it what was needed?
> > ```
> > // nbBiere = 1 ➔ "Ich hatte nur 1 Bier, Liebling, ich schwöre"
> > // nbBiere = 2 ➔ "Ich hatte nur 2 Bären, Liebling, ich schwöre"
> > // nbBiere = 6 ➔ "Ich hatte nur 6 Bären, Liebling, ich schwöre"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > > I suggested a few other changes, then it should be good to merge.
> 
> The spelling mistake must be on a different word than beer, in this part of the sentence: "Liebling, ich schwöre".

Ah, got it

**Pull request merged**